__tests__/pubsub.service.google.test.ts

```diff
@@ -18,35 +18,40 @@ const mockAllUsersList = [
 jest.mock('../src/client/googlePubSub', () => ({
   __esModule: true,
   default: class {
-    public static getInstance(): any {
+    public static getInstance() {
       return new this();
     }
-    public publish(...args: any[]): any {
+    public publish(...args: any[]) {
       return mockPublish(...args);
     }
-    public subscribe(...args: any[]): any {
+    public subscribe(...args: any[]) {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8106467)
```suggestion
    public subscribe(...args: unknown[]) {
```